docs/testcases.md

```diff
@@ -1,396 +1,35 @@
-
-(Optional) Wait for new pushes to that repo.
-
-**Expected**: pushes loaded should continue to honor that range.
-
-## Perfherder Graphs
-
-Load Perfherder at eg.
-<https://treeherder.allizom.org/perf.html>
-
-**Expected**: Landing page should appear.
-
-Click the blue 'Add test data' button, select a platform, enter a test series, and click Add+.
-
-**Expected**: Performance series should load with scatter graph and line graph.
-
-Click Add more test data, and add a 2nd series.
-
-**Expected**: The second series is drawn in an alternate color, and both series can have their displays disabled/enabled via Show/Hide series tick UI.
-
-Change display range dropdown to 90 days (or other value)
-
-**Expected**: Ensure both series expand to that date range. Confirm the data which has expired beyond the 6 week data cycle still appears, but the SHA just will instead display "loading revision".
-
-## No console errors throughout test run
-
-Ensure the browser console is error free during and after the test run.
-Open the console during the test run.
-
-**Expected**: No errors should appear in the console.
-
-## Perfherder Compare
-
-Load Perfherder Compare at eg.
-<https://treeherder.allizom.org/perf.html#/comparechooser>
-
-**Expected**: Landing page should appear.
-
-Select two push revisions from the 'Recent' dropdowns, and click 'Compare revisions'.
-
-**Expected**: Some kind of result should appear (likely a warning "tests with no results: " table).
-
-Click on the 'Substests' link for a row.
-
-**Expected**: Sub-compare results should appear.
-
-Click on the 'Graph' link for a sub-compare row if it exists.
-
-**Expected**: The plotted graph for that series should appear.
-
-## All keyboard shortcuts
-
-Check all keyboard shortcut functionality as listed in the [userguide].
-
-**Expected**: Each shortcut should work as expected.
-
-## Job counts
-
-In any push with job counts, click on the group button eg. B( ) to expand the count.
-
-**Expected**: Jobs should appear.
-
-Select an expanded job, and click again on the group button B() to collapse the count back down.
-
-**Expected**: The count should appear as a highlighted large button. eg. pending gray "+14"
-
-Click in empty space to deselect the collapsed job.
-
-**Expected**: The count "+14" should be deselected.
-
-Click on the ( + ) global Expand/Collapse icon in the navbar to toggle all +n counts.
-
-**Expected**: Counts should expand and collapse on all visible pushes.
-
-Navigate via the n,p and left/right keys.
-
-**Expected**: +n counts should be skipped during navigation.
-
-expand all the groups, (the url querystring will reflect this) then reload the page
-
-**Expected**: groups should still be expanded for all pushes
-
-Optional: There are other variants that can be tested: classification of expanded job count members, Filters, and any other workflow integration testing.
-
-[userguide]: https://treeherder.mozilla.org/userguide.html
+1. Ask in `#sheriffs` to test their workflows.
```

Comment:
I normally just use the word "sheriff" on the Treeherder channel to get their attention. It might be better in #treeherder since we can keep conversations around deployments in the same place.

Reply:
Oh interesting, I didn't know that would work to get their attention.  :). Good tip!